### PR TITLE
Add Angular, Vue and React code snippets to dxMenu

### PR DESCRIPTION
**Pull request by EugeniyKiyashko**

### CODE REVIEW

File: concepts/05 Widgets/Menu/03 Access the Clicked Item.md [hunk 1]
```diff
@@ -41,16 +39,69 @@ To access the clicked item, handle the [itemClick](/api-reference/10%20UI%20Widg
...
+
+    import DxMenu from 'devextreme-vue/menu';
+
+    export default {
+        components: {
+            DxMenu
+        },
+        methods: {
+            itemClick(e) {
+                const { itemData, itemElement, itemIndex } = e;
```
Comment (AlexJameson) [inline concepts/05 Widgets/Menu/03 Access the Clicked Item.md:62]: Do we assign all the [function parameter's fields](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxMenu/Configuration/#onItemClick) to these consts?

File: concepts/05 Widgets/Menu/03 Access the Clicked Item.md [hunk 2]
```diff
@@ -41,16 +39,69 @@ To access the clicked item, handle the [itemClick](/api-reference/10%20UI%20Widg
...
 If you are going to change event handlers at runtime, or if you need to attach several handlers to the **itemClick** event, subscribe to this event using the [on(eventName, eventHandler)](/api-reference/10%20UI%20Widgets/Component/3%20Methods/on(eventName_eventHandler).md '/Documentation/ApiReference/UI_Widgets/dxMenu/Methods/#oneventName_eventHandler') method. This approach is more typical of jQuery.
 
     <!--JavaScript-->
-    var itemClickHandler1 = function (e) {
+    var itemClickHandler1 = function(e) {
         // First handler of the "itemClick" event
     };
 
-    var itemClickHandler1 = function (e) {
+    var itemClickHandler1 = function(e) {
```
Comment (AlexJameson) [inline concepts/05 Widgets/Menu/03 Access the Clicked Item.md]:
```suggestion
    var itemClickHandler2 = function(e) {
```

File: concepts/05 Widgets/Menu/05 Customize Item Appearance.md [hunk 3]
```diff
@@ -98,104 +177,155 @@ If you need a more flexible solution, define a custom template. For Angular, Ang
...
+
 You can also customize an individual menu item. For this purpose, declare a template for this item as a script and pass its `id` to the [template](/api-reference/_hidden/CollectionWidgetItem/template.md '/Documentation/ApiReference/UI_Widgets/dxMenu/Configuration/items/#template') field of the item's data object.
 
-    <!--HTML--><script id="individualTemplate" type="text/html">
-        <!-- ... -->
+---
+
+##### jQuery
+
+    <!--HTML-->
```
Comment (AlexJameson) [inline concepts/05 Widgets/Menu/05 Customize Item Appearance.md:307]: Let's add a div with id="menuContainer"

File: concepts/05 Widgets/Menu/10 Change the Orientation.md [hunk 4]
```diff
@@ -74,6 +121,50 @@ When the widget is positioned at the bottom or at the left side, you may want to
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxMenu ...
+            submenuDirection="rightToTop"
```
Comment (AlexJameson) [inline concepts/05 Widgets/Menu/10 Change the Orientation.md]:
```suggestion
            submenu-direction="rightToTop"
```

**Review by AlexJameson — CHANGES_REQUESTED** [review]

File: concepts/05 Widgets/Menu/03 Access the Clicked Item.md (hunk 1 above)
Comment (EugeniyKiyashko) [reply concepts/05 Widgets/Menu/03 Access the Clicked Item.md:62]: In this example doesn't have to

File: concepts/05 Widgets/Menu/05 Customize Item Appearance.md [hunk 5]
```diff
@@ -98,104 +177,159 @@ If you need a more flexible solution, define a custom template. For Angular, Ang
...
+    <div id="menuContainer"></div>
```
Comment (AlexJameson) [inline concepts/05 Widgets/Menu/05 Customize Item Appearance.md]:
```suggestion

```

**Review by AlexJameson — APPROVED** [review]

**Pull request merged**